packages/xo-server/src/_pDebounceWithKey.spec.js

```diff
@@ -0,0 +1,22 @@
+/* eslint-env jest */
+
+import { debounceWithKey, REMOVE_CACHE_ENTRY } from './_pDebounceWithKey'
+
+test('clear the cache of debounced function', async () => {
+  const key = 2
+
+  let i = 0
+  const debouncedFn = debounceWithKey(
+    function() {
+      return Promise.resolve(++i)
+    },
+    120e3,
+    id => id
+  )
+
+  await expect(debouncedFn(key)).resolves.toBe(1)
```

Comment:
```suggestion
  expect(await debouncedFn(key)).toBe(1)
```

This is simpler and less Jest specific.